### PR TITLE
[PaymentBundle] Form registration, missing specs and CS Fixes.

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Component/Payment/Repository/PaymentMethodRepositoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * @author Arnaud Langlade <arn0d.dev@gmail.com>
+ */
+interface PaymentMethodRepositoryInterface
+{
+    /**
+     * @return array
+     */
+    public function getQueryBuidlerByStatus($disabled);
+}
```
Comment (Shine-neko) [inline src/Sylius/Component/Payment/Repository/PaymentMethodRepositoryInterface.php]: Eof ?

Comment (aRn0D) [inline src/Sylius/Component/Payment/Repository/PaymentMethodRepositoryInterface.php]: Done

**Comment by pjedrzejewski** [comment]
@aRn0D Could you have a look at the failing build?

**Comment by aRn0D** [comment]
@pjedrzejewski yes I will have a have a look this week !

**Comment by aRn0D** [comment]
@pjedrzejewski fixed, let's see the build

**Comment by michalmarcinkowski** [comment]
:+1:

**Comment by pjedrzejewski** [comment]
Thank you Arnaud! :)

**Comment by umpirsky** [comment]
One note here. This PR is a BC break, I have added it to https://github.com/Sylius/Sylius/pull/3393.

`PaymentMethodEntityType` is removed, and we used to override this class.

**Pull request merged**